tron-transactions/utils.go

```diff
@@ -0,0 +1,11 @@
+package trontransactions
+
+import (
+	"regexp"
+)
+
+func isFilterCorrect(s string) bool {
+	// Regular expression: Allows lowercase letters, numbers, and underscores, separated by commas
+	re := regexp.MustCompile(`^[a-z0-9_]+(,[a-z0-9_]+)*$`)
```

Comment:
Move to package variable, no need to recompile a regex each time, can be re-used.